product-backend/scripts/example_data.sql

```diff
@@ -0,0 +1 @@
+-- TODO

```

Comment:
without USE <DBNAME> here or dbname setup in shell script this may not work; Please add at least for now even a simple query like show databases with a comment that it's to be removed with first meaningful code.